example-runner/src/main/java/uk/gov/gchq/palisade/example/runner/runner/RestExample.java

```diff
@@ -56,14 +56,15 @@ private void makeRequest(final String userId, final String fileName, final Strin
      * @param args command-line arguments
      * @throws IOException for any file system error
      */
+    @Override
     public void run(final String... args) throws IOException {
-        final String alice = "Alice";
-        final String bob = "Bob";
-        final String eve = "Eve";
+        String alice = "Alice";
+        String bob = "Bob";
+        String eve = "Eve";
 
-        final String salary = Purpose.SALARY.name();
-        final String dutyOfCare = Purpose.DUTY_OF_CARE.name();
-        final String staffReport = Purpose.STAFF_REPORT.name();
+        String salary = Purpose.SALARY.name();
+        String dutyOfCare = Purpose.DUTY_OF_CARE.name();
+        String staffReport = Purpose.STAFF_REPORT.name();
```

Comment:
Is there a reason why these are no longer `final`?

Reply:
SonarQube disagrees with Checkstyle over SNAKE_CAPS or camelCase if `final`